doc/developing_on_windows.md

```diff
@@ -49,9 +49,11 @@ pacman, so have a look to see what works for you. Both tools can be
 installed with the commands:
 
 ```
-stack exec -- pacman -S mingw-w64-x86_64-make
-stack exec -- pacman -S mingw-w64-x86_64-cmake
+stack exec -- pacman -- -S mingw-w64-x86_64-make
```

Comment:
Not only do I not need to run the command this way on my Windows machine, but the new command fails (for both cmd.exe and PowerShell):

```
>stack exec -- pacman -- -S mingw-w64-x86_64-make
error: no operation specified (use -h for help)
```

Reply:
strange. This is the output of my powershell console.
```
source/Other/stack-project via ⬢ v12.18.1 took 3s at 21:48:38
❯ $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19041
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

source/Other/stack-project via ⬢ v12.18.1 at 21:48:43
❯ stack exec -- pacman -S mingw-w64-x86_64-make
Invalid option `-S'

Usage: stack.exe exec CMD [-- ARGS (e.g. stack exec -- ghc-pkg describe base)]
                      [--[no-]ghc-package-path] [--[no-]stack-exe]
                      [--package ARG] [--rts-options RTSFLAG] [--cwd DIR]
                      [--setup-info-yaml URL] [--help]
  Execute a command
source/Other/stack-project via ⬢ v12.18.1 at 21:48:57
❯ stack exec -- pacman -- -S mingw-w64-x86_64-make
resolving dependencies...
looking for conflicting packages...

Packages (9) mingw-w64-x86_64-expat-2.2.5-1  mingw-w64-x86_64-gcc-libs-7.3.0-2  mingw-w64-x86_64-gettext-0.19.8.1-3  mingw-w64-x86_64-gmp-6.1.2-1
             mingw-w64-x86_64-libiconv-1.15-2  mingw-w64-x86_64-libwinpthread-git-6.0.0.5134.2416de71-1  mingw-w64-x86_64-mpc-1.1.0-1  mingw-w64-x86_64-mpfr-4.0.1-1
             mingw-w64-x86_64-make-4.2.1-2

Total Download Size:    5.50 MiB
Total Installed Size:  27.09 MiB

:: Proceed with installation? [Y/n] n
```

Let's wait and see whether anyone else have this issue.